practice/spec/regex/regex_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe 'Regex pattern' do
+  before(:each) do
+    @book_name = 'Story.txt'
+    @file_name = 'alphanumeric.txt'
+  end
+
+  it 'should not found not existing string' do
+    matches = RegexPatterns.count_occurrence(@book_name, 'Can not be in book')
```

Comment:
We should have this string in variable in case if we want to use one more time in a future. It should be a constant because we aren't going to modify it.
```
NOT_EXISTING_STRING = 'Can not be in book'.freeze
```
Also, you can put it before *before* hook